src/http_rest_api_handler.cpp

```diff
@@ -568,6 +568,24 @@ void HttpRestApiHandler::convertShapeType(Value& scope, Document& doc) {
     }
 }
 
+void HttpRestApiHandler::convertRTInfo(Value& scope, Document& doc, ov::AnyMap& rt_info) {
+    scope.SetObject();
+    for (auto& [key, value] : rt_info) {
+        SPDLOG_DEBUG("building rest response: rt_info: key: {}; value: {}", key, value.as<std::string>());
+        rapidjson::Value rt_info_key, rt_info_value, subScope;
+        rt_info_key.SetString(key.c_str(), doc.GetAllocator());
+        if (value.is<ov::AnyMap>()) {
+            SPDLOG_INFO("building submap rest response : key: {};", key);
+            subScope.SetObject();
+            convertRTInfo(subScope, doc, value.as<ov::AnyMap>());
+            scope.AddMember(rt_info_key, subScope, doc.GetAllocator());
+        } else {
+            rt_info_value.SetString(value.as<std::string>().c_str(), doc.GetAllocator());
```

Comment:
Are you sure that if value is not ov::AnyMap it **must** be cast-able to std::string? I think it's risky, we should at least try catch it so we dont end up in undefined behavior